integration_tests/requirements.txt

```
@@ -1,6 +1,6 @@
-dbt-snowflake~=1.0.0
-dbt-bigquery~=1.0.0
-dbt-redshift~=1.0.0
-dbt-postgres~=1.0.0
-dbt-spark~=1.0.0
-dbt-spark[PyHive]~=1.0.0
+dbt-snowflake~=1.0.0 
+dbt-bigquery~=1.0.0 
+dbt-redshift~=1.0.0 
+dbt-postgres~=1.0.0 
+dbt-spark~=1.0.0 
+dbt-spark[PyHive]~=1.0.0 
```

We want the requirements to pull the latest versions of dbt.

Can you change these to be the following:
```suggestion
dbt-snowflake>=1.0.0 
dbt-bigquery>=1.0.0 
dbt-redshift>=1.0.0 
dbt-postgres>=1.0.0 
dbt-spark>=1.0.0 
dbt-spark[PyHive]>=1.0.0 
```